app/views/pages/index.html.erb

```diff
@@ -3,3 +3,4 @@
   <p>Find me in app/views/pages/index.html.erb</p>
 </div>
 
+
```

Comment:
we added 2 lines at the end of the file?

Reply:
touqer bhai advised to set one empty line of each file to avoid conflicts

Reply:
but there're two line, aren't they?